verify-targets.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
```

Comment:
Protip: If you add add a shebang to a `.js` file, you may as well make it executable and then invoke it simply with `./verify-targets.js`. :wink: